packages/expo-screen-orientation/README.md

```diff
@@ -0,0 +1,18 @@
+# expo-screen-orientation
+
+Allows you to manage your device's orientation.
+
+# API documentation
+
+- [Documentation for the master branch](https://github.com/expo/expo/blob/master/docs/pages/versions/unversioned/sdk/screen-orientation.md)
+- [Documentation for the latest stable release](https://docs.expo.io/versions/latest/sdk/screen-orientation/)
+
+# Installation
+
+This package is pre-installed in [managed](https://docs.expo.io/versions/latest/introduction/managed-vs-bare/) Expo projects. You may skip the rest of the installation guide if this applies to you.
```

Comment:
This package is not pre-installed. People will need to do `expo install expo-screen-orientation`...